packages/front-end/components/Experiment/Templates/TemplatesPage.tsx

```diff
@@ -96,6 +97,20 @@ export const TemplatesPage = ({
     return <div className="alert alert-danger">{error.message}</div>;
   }
 
+  if (!hasTemplatesFeature) {
+    return (
+      <>
+        <NoAccessState
```

Comment:
I would rename this component.  Something like `PremiumEmptyState`.  We use "no access" to mean something different in the code base (a user role without read access).

Reply:
Fixed